lib/origin_simulator/payload.ex

```diff
@@ -26,11 +26,9 @@ defmodule OriginSimulator.Payload do
 
   def body(_server, status, path \\ Recipe.default_route(), route \\ Recipe.default_route()) do
     case {status, path} do
-      {200, _} -> cache_lookup(route)
-      {404, _} -> {:ok, "Not found"}
       {406, "/*"} -> {:ok, OriginSimulator.recipe_not_set()}
       {406, _} -> {:ok, OriginSimulator.recipe_not_set(path)}
-      _ -> {:ok, "Error #{status}"}
+      _ -> cache_lookup(route)
```

Comment:
Sorry maybe I'm missing the reasoning for this. I'm not sure why it's trying to fetch from cache for 404s

Reply:
ah it's probably the discussion in Slack. It'd be useful to add a line in the PR desc